NewsML-G2 v2.34

bquinn opened the pull request:
Changes in version 2.34:

* http://dev.iptc.org/G2-CR00218-increase-nesting-of-related (fixes iptc/news-architecture-dev#144)
* http://dev.iptc.org/G2-CR00219-add-datamining-element-to-rightsinfo (fixes iptc/news-architecture-dev#136)
* Updating dev schema to be based on version 2.34. Fixes iptc/news-architecture-dev#146

bquinn committed:
Adding another test case for 2.33

bquinn committed:
Setting up ready for NewsML-G2 v2.34

bquinn committed:
Adding a third level of <related> nesting

bquinn committed:
Updating dev schema to be based on version 2.34. Fixes iptc/news-architecture-dev#146

davecompton commented:
Hi @bquinn - just checking: So have all the CR00218 & CR00219 changes been applied for this pull - looks like so, from the above. Thanks.

bquinn commented:
> Hi @bquinn - just checking: So have all the CR00218 & CR00219 changes been applied for this pull - looks like so, from the above. Thanks.

Actually no, I still need to commit the dataMining changes. I'm trying to get the RightsML example to validate properly.

bquinn committed:
Implemented <dataMining> element as agreed at NAR call on 23rd Feb.

bquinn commented:
> > Hi @bquinn - just checking: So have all the CR00218 & CR00219 changes been applied for this pull - looks like so, from the above. Thanks.
> 
> Actually no, I still need to commit the dataMining changes. I'm trying to get the RightsML example to validate properly.

It's working and committed now! Please take a look.

davecompton commented on specification/NewsML-G2_2.34-spec-All-Power.xsd:
Still shows v2.33 in comments?

davecompton commented on tests/schema_versions/NewsML-G2_2.34-spec-All-Power.xsd:
Still shows v2.33 in comments?

bquinn committed:
Fixing version number in README and XML schema files

bquinn commented:
good point! I just fixed it.

On Thu, Feb 29, 2024 at 3:46 PM Dave Compton ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> On tests/schema_versions/NewsML-G2_2.34-spec-All-Power.xsd
> <https://github.com/iptc/newsml-g2/pull/19#discussion_r1507609042>:
>
> Still shows v2.33 in comments?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/iptc/newsml-g2/pull/19#pullrequestreview-1908928488>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACLVWDWXKL3IS2C3NNUT3DYV4YJXAVCNFSM6AAAAABDWXTPCCVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTSMBYHEZDQNBYHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

davecompton commented on specification/NewsML-G2_2.34-spec-All-Power.xsd:
Still shows: Date of creation of this XML Schema document revision: 2023-08-25

davecompton commented on tests/schema_versions/NewsML-G2_2.34-spec-All-Power.xsd:
Still shows: Date of creation of this XML Schema document revision: 2023-08-25

bquinn committed:
Updated XML Schema component files and examples as per our maintenance process.
Resulting schema is the same as the one created previously except that the dates
are now correct for version 2.34.

bquinn committed:
Change references to "MIME type" to "Media Type". Fixes #145.

bquinn committed:
Release files for 2.34. XML Spy schema docs not yet updated.

davecompton commented on releases/2.34/specification/XML-Schema-Doc-Power/NewsML-G2_2.33-spec-All-Power.html:
Filenames show 2.33.

bquinn replied:
Yep I haven't updated the XML Spy schema docs yet. I mentioned that in a commit message but github might have buried it somewhere.

davecompton commented on examples/LISTING_4_Multiple_Renditions_of_a_Video_in_NewsML-G2.xml:
Odd ^M in this file?

bquinn replied:
I think that has been around for a long time. Fixed now.

bquinn committed:
Changed line-endings of an example file, resolving comment from Dave in the PR.

bquinn committed:
XML Schema docs and final update for 2.34 release.
